src/screens/OrgList/OrgList.test.tsx

```diff
@@ -402,77 +418,6 @@ describe('Organisation List Page', () => {
   });
 });
 
-test('Search bar filters organizations by name', async () => {
```

Comment:
Why was this test removed?

Reply:
@Rumone 

Manually the search bar is functional right now, but the tests seem to break after the changes. I have created a separate issue to fix it up.

Reply:
Oh I saw that you mentioned that my bad.